aws/ecr/kops_ecr_app.tf

```diff
@@ -3,14 +3,14 @@ variable "kops_ecr_app_repository_name" {
 }
 
 module "kops_ecr_app" {
-  source       = "git::https://github.com/cloudposse/terraform-aws-kops-ecr.git?ref=tags/0.1.4"
+  source       = "git::https://github.com/cloudposse/terraform-aws-kops-ecr.git?ref=use-roles-instead-of-users"
```

Comment:
Pin to release

Reply:
```suggetion
  source       = "git::https://github.com/cloudposse/terraform-aws-kops-ecr.git?ref=tags/0.1.4"
```

Reply:
It's currently pinned to `use-roles-instead-of-users`